File: scrumble-desktop/ScrumbleLib/Data/TaskState.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ScrumbleLib.Data
{
    public enum TaskState
    {
        PRODUCT_BACKLOG,
        SPRINTBACKLOG,
        IN_PROGRESS,
        TO_VERIFY,
        DONE
    }

    public static class TaskStateParser
    {
        public static TaskState Parse(string str)
        {
            return (TaskState)Enum.Parse(typeof(TaskState), str);
        }
    }
}

File: scrumble-android/app/src/main/java/com/spogss/scrumble/enums/TaskState.kt
package com.spogss.scrumble.enums

enum class TaskState {
    PRODUCT_BACKLOG,
    SPRINT_BACKLOG,
    IN_PROGRESS,
    TO_VERIFY,
    DONE;

    fun getShortForm(): String {
        return when(this) {
            PRODUCT_BACKLOG -> "PRBL"
            SPRINT_BACKLOG -> "SPBL"
            IN_PROGRESS -> "PROG"
            TO_VERIFY -> "TEST"
            DONE -> "DONE"
        }
    }
}